src/sentry/utils/email.py

```diff
@@ -389,3 +392,26 @@ def send_mail(subject, message, from_email, recipient_list, fail_silently=False)
         subject, message, from_email, recipient_list,
         connection=get_connection(fail_silently=fail_silently),
     )
+
+
+class PreviewBackend(BaseEmailBackend):
+    """
+    Email backend that can be used in local development to open messages in the
+    local mail client as they are sent.
+
+    Probably only works on OS X.
+    """
+    def send_messages(self, email_messages):
+        for message in email_messages:
+            content = str(message.message())
+            preview = tempfile.NamedTemporaryFile(
```

Comment:
Would we wanna be safe and do:

```python
preview = tempfile.NamedTemporaryFile(
    delete=False,
    prefix='sentry-email-preview-',
    suffix='.eml',
)
try:
    preview.write(content)
    preview.flush()
finally:
    preview.close()
```

To make sure a fd doesn't get left behind.